test/ts/unit/multisig_wallet.ts

```diff
@@ -0,0 +1,94 @@
+// External
+import { BigNumber } from "bignumber.js";
+import * as chai from "chai";
+import * as Web3 from "web3";
+import * as Units from "../test_utils/units";
+
+// Contracts
+import { MultiSigWalletContract } from "../../../types/generated/multi_sig_wallet";
+
+// Test utils
+import { BigNumberSetup } from "../test_utils/bignumber_setup";
+import ChaiSetup from "../test_utils/chai_setup";
+import { INVALID_OPCODE, REVERT_ERROR } from "../test_utils/constants";
+
+// Configure BigNumber exponentiation
+BigNumberSetup.configure();
+
+// Set up Chai
+ChaiSetup.configure();
+const expect = chai.expect;
+
+const MultiSigWallet = artifacts.require("MultiSigWallet");
+
+contract("Multisig Wallet (Unit Tests)", (ACCOUNTS) => {
+    let wallet: MultiSigWalletContract;
+
+    const CONTRACT_OWNER = ACCOUNTS[0];
+    const DESTINATION = ACCOUNTS[1];
+    const TX_DEFAULTS = { from: CONTRACT_OWNER, gas: 4000000 };
+
+    before(async () => {
+        const walletTruffle = await MultiSigWallet.new([CONTRACT_OWNER], new BigNumber(1), {
+            from: CONTRACT_OWNER,
+        }); // truffle
+        const walletWeb3 = web3.eth.contract(walletTruffle.abi).at(walletTruffle.address); // web3
```

Comment:
I'm not a fan of inline comments like this

Reply:
i think of them as hints. not comments.